src/hooks/useCurrentMarketId.ts

```diff
@@ -131,10 +133,17 @@ export const useCurrentMarketId = () => {
       abacusStateManager.setMarket(DEFAULT_MARKETID);
       abacusStateManager.setTradeValue({ value: null, field: null });
     } else if (hasMarketIds) {
-      abacusStateManager.setMarket(marketId ?? DEFAULT_MARKETID);
+      if (marketId) {
+        if (hasMarketOraclePrice) {
```

Comment:
its not immediately obvious why oraclePrice is the benchmark for being able to load things for abacus

Reply:
This is what I said last time! This variable name CONFUSES me

Reply:
It’s not T.T there is actually a 30 sec cadence where the Indexer updates its internal list of markets before orderbook subscription is possible. Looking into how we can make this work

Reply:
Will follow up with renaming the vars after we figure out how to successfully subscribe to the orderbook